helm/polaris/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,36 @@
+{{/*
+  Copyright (c) 2024 Snowflake Computing Inc.
```

Comment:
nit: why do we have a copyright header in this YAML file, but not in others? Are copyright headers required in Helm for Polaris?

Reply:
Good catch on this one. So @flyrain suggested we should have license header for all files we check-in, thus I added them in. After I reviewed the official helm chart best practice and having a separate `LICENSE` file seems to be more suitable for helm chart file. I then removed the license header from all files that I added. This one was a miss. This is removed now.